client/src/components/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import ReviewList from './ReviewList.jsx'
+import ReviewList from './ReviewList.jsx';
+import RelatedContainer from './RelatedContainer.jsx';
 
 class App extends React.Component {
```

Comment:
This is a general comment - if we're using hooks, should we keep the App component as a functional component as well?

Reply:
Lols didnt think about that Maybe????